R/roxygen.R

```diff
@@ -21,6 +21,8 @@ use_roxygen_md <- function() {
     if (!uses_git()) {
       ui_todo("Use git to ensure that you don't lose any data")
     }
+
+    check_installed("roxygen2md")
```

Comment:
Just below ... does "existing Rd markdown" even make sense? Feels like we could word that better.

Reply:
Yeah, good catch.